apiserver/params/status.go

```diff
@@ -231,6 +231,11 @@ type StatusHistoryPruneArgs struct {
 	MaxHistoryMB   int           `json:"max-history-mb"`
 }
 
+type ActionPruneArgs struct {
```

Comment:
If the rest of the ActionPrune code is with action and not statushistory, perhaps this struct belongs there too?